app/src/main/res/values/strings.xml

```diff
@@ -1138,6 +1138,10 @@
     <string name="sign_in_with_camera">Sign in with your camera</string>
     <!-- Text shown for settings option for sign with email -->
     <string name="sign_in_with_email">Use email instead</string>
+    <!-- Text shown for settings option for sign with email -->
```

Comment:
nit - let's add to the string description that Firefox is intentionally hardcoded?

Reply:
Also I don't think these strings descriptions are right? 🤔 Let's update those too!

Reply:
Oh and we probably also want to add what the substitution in the string is about generally